pandas/_libs/src/headers/cmath

```diff
@@ -0,0 +1,13 @@
+#ifndef _PANDAS_MATH_H_
+#define _PANDAS_MATH_H_
+
```

Comment:
can you add commets here on why we have this file (so the next person doesn't go thru the same as you :>)